en/docs/how-tos/endpoints/overriding-endpoint-information/overriding-endpoints-for-imported-apis.md

```diff
@@ -3,55 +3,53 @@
 !!! note
     This information is only applicable for APIs imported from WSO2 API Manager.
 
-In the event, a backend service connection URL is added or edited, the backend connection URL provided in the respective OpenAPI definition needs to be overriden. The following section describes how this can be overriden on the APIs that are imported from the WSO2 API Manager.
+The API imported from the API Manager once deployed in Microgateway, it needs to be moved across different environments. When moving from one environment
+to the other backend service details can alter. In those cases, we need to override the endpoint details of the API
+The following section describes how this can be overriden on the APIs that are imported from the WSO2 API Manager.
 
-#### Overriding endpoints at runtime
+## Overriding endpoints at runtime
 
 Use the following command to override the endpoints of an API imported from the WSO2 API Manager at server start-up.
 
 Follow the instructions below to override the production or sandbox endpoint(s).
 
 1.  Use the following command to start WSO2 Microgateway. Here the system variables are provided at server start-up to override the endpoint of the API, which has been added to WSO2 API Microgateway using an OpenAPI definition.
 
-    ``` text tab="Override single endpoint"
+    ``` text tab="Override endpoints - binary"
     gateway <path-to-MGW-jar> 
-    --<API-ID>_<endpoint-type>_endpoint_<endpoint-index>="<endpoint_URL>"
+        --<API-ID>_<endpoint-type>_endpoint_<endpoint-index>="<endpoint-URL-1>" 
+        --<API-ID>_<endpoint-type>_endpoint_<endpoint-index>="<endpoint_URL_2>"
     ```
     
-    ``` text tab="Override Multiple endpoints"
-    gateway <path-to-MGW-jar> 
-    --<API-ID>_<endpoint-type>_endpoint_<endpoint-index>="<endpoint-URL-1>" 
-    --<API-ID>_<endpoint-type>_endpoint_<endpoint-index>="<endpoint_URL_2>"
+    ``` text tab="Override endpoints - docker"
+    docker run -d -p 9090:9090 -p 9095:9095 -e <API-ID>_<endpoint-type>_endpoint_<endpoint-index>="<endpoint_URL-1>" -e <API-ID>_<endpoint-type>_endpoint_<endpoint-index>="<endpoint_URL-2>" <docker_image_name>
     ```
     
-    - `<API-ID>` - ID generated for the imported API. You can find it by checking the file name of imported API located in `<project>/gen/api_definitions/`
+    - `<API-ID>` - ID generated for the imported API. You can find it by checking the file name of imported API located in `<PROJECT_HOME>/gen/api_definitions/`. Sample file name would be `api_30e623704c5c5479b7c0d9ab78e965df02c1610401e37cbd557e6353e3191c76swagger.json`. Here the 
+    `<API-ID>` is `api_30e623704c5c5479b7c0d9ab78e965df02c1610401e37cbd557e6353e3191c76`
```

Comment:
we can move this inside the format `text tab="Format - binary/docker"`. Because most of our docs explain variables like that.